CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog for @pdc/service
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
Given that this is an incompatible change, we would expect a major version update, right? That or we should not mention we follow semver.